extensions/gif-search/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # GIF Search Changelog
 
+## [Show "Recent" GIFs]
```

Comment:
This makes it show up properly in the in-app changelog.

```suggestion
## [Show "Recent" GIFs] - 2022-04-13
```

Reply:
Whoops! Good catch

Reply:
@thomaspaulmann done!